gtk/src/lib.rs

```diff
@@ -16,71 +16,71 @@
 //!
 //! See also:
 //!
-//! - [Gtk-rs documentation overview](https://gtk-rs.org/docs-src/)
+//! - [gtk-rs documentation overview](https://gtk-rs.org/docs-src/)
 //!
 //! - [General `GLib` family types and object system overview](../glib/index.html)
 //!
-//! - [GTK+ documentation](https://www.gtk.org/docs/)
+//! - [GTK documentation](https://www.gtk.org/docs/)
 //!
-//! # Hello World
+//! ## “Hello, World!” example program
```

Comment:
There is no need for the "fancy" (don't know what they're called) double-quote, rustdoc is changing them automatically (as you can see [here](https://doc.rust-lang.org/nightly/std/primitive.never.html)).